t/errlog.t

```diff
@@ -9,21 +9,32 @@ log_level('error');
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 2 + 11);
+plan tests => repeat_each() * (blocks() * 2 + 12);
 
 my $pwd = cwd();
 
 add_block_preprocessor(sub {
     my $block = shift;
 
+    my $test_name = $block->name();
     my $http_config = $block->http_config || '';
-    $http_config .= <<_EOC_;
 
-    lua_package_path "$pwd/lib/?.lua;../lua-resty-lrucache/lib/?.lua;;";
-    init_by_lua_block {
-        require "resty.core"
+    if (index($test_name, "TEST 25: ") != -1) {
```

Comment:
@membphis 
A separate test file for a single test seems wasteful...
I am not familiar with `test-nginx`. Is there any pretty solution?